### DISCUSSION TITLE
Question about Proposer Selection

### DISCUSSION THREAD

**Discussion by nashqueue** [tendermint/tendermint#9328] [Q&A]
Hi guys, I have a question about tendermints proposer selection.
I am currently reading https://github.com/tendermint/tendermint/blob/v0.34.x/spec/consensus/proposer-selection.md#new-validator and the validator_set.go codebase. 

My question is regarding the penalty factor of -1.125 when a new validator is added. 

In my case I want to add a validator with a large portion of VP as in the docs example.
Could I circumvent the penalty factor by creating a new validator with very little VP and then in the next round increase the voting power to the desired one? 

In the example of the docs p3 is added with a VP of 8 and gets a penalty of -13.  I would try to circumvent this by adding p3 with a VP close to zero and therefore the penalty is only -4.5.  

I guess my question is what happens when I change the VP after I already added the validator?

**Comment by cmwaters** [comment]
Hey, is your aim in doing this to make the new validator the proposer sooner?
